sonar-core/src/main/java/org/sonar/core/platform/RemotePlugin.java

```diff
@@ -25,6 +25,11 @@
 import org.apache.commons.codec.digest.DigestUtils;
 import org.apache.commons.lang.StringUtils;
 
+/**
+ * @deprecated since 6.6 Used for deprecated deploy/plugin/index.txt
+ *
+ */
+@Deprecated
 public class RemotePlugin {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)